### PR TITLE
Card pages for [ Hacktoberfest ]

### PR DESCRIPTION
**Pull request by haryanapnx**

### CODE REVIEW

File: src/views/Card.vue [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<template>
+  <div>
+    <h3 class="text-gray-700 text-3xl font-semibold">Card</h3>
+
+    <div class="mt-4 mb-3">
+      <h4 class="text-gray-700">Stacked</h4>
+
+      <div class="max-w-sm rounded overflow-hidden shadow-lg mt-6">
```
Comment (Citrullin) [inline src/views/Card.vue]: You should add `bg-white` to the class.

File: src/views/Card.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        <img
+          class="w-full"
+          src="https://via.placeholder.com/384x234"
+          alt="Sunset in the mountains"
+        />
+        <div class="px-6 py-4">
+          <div class="font-bold text-xl mb-2">The Coldest Sunset</div>
```
Comment (Citrullin) [inline src/views/Card.vue]: I would add `text-gray-900` here for consistency with the other pages.

File: src/views/Card.vue [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  <div>
+    <h3 class="text-gray-700 text-3xl font-semibold">Card</h3>
+
+    <div class="mt-4 mb-3">
+      <h4 class="text-gray-700">Stacked</h4>
+
+      <div class="max-w-sm rounded overflow-hidden shadow-lg mt-6">
+        <img
+          class="w-full"
+          src="https://via.placeholder.com/384x234"
```
Comment (Citrullin) [inline src/views/Card.vue]: Would probably look better, if you use some nice stock image here.

File: src/views/Card.vue [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      </div>
+    </div>
+    <hr />
+    <div class="mt-5">
+      <h4 class="text-gray-700">Horizontal</h4>
+
+      <div class="max-w-sm w-full lg:max-w-full lg:flex mt-6">
+        <div
+          class="h-48 lg:h-auto lg:w-48 flex-none bg-cover rounded-t lg:rounded-t-none lg:rounded-l text-center overflow-hidden"
+          style="background-image: url('https://via.placeholder.com/192x291')"
```
Comment (Citrullin) [inline src/views/Card.vue]: Same as the other: Stockimage looks better.

**Comment by haryanapnx** [comment]
I have revised the above issue, please check again @Citrullin

**Comment by haryanapnx** [comment]
can you merge @wobsoriano @suriarte-orangeloops ?

**Comment by wobsoriano** [comment]
I wasn't actually looking for additional pages but okay I will merge this one and not the 404 page. Thanks.

**Pull request merged**